services/workflows-service/src/data-analytics/data-analytics.service.ts

```diff
@@ -84,7 +77,7 @@ export class DataAnalyticsService {
 
     if (!isEmpty(transactionType)) {
       conditions.push(
-        Prisma.sql`tr."transactionType"::text IN (${Prisma.join(transactionType, ',')})`,
+        Prisma.sql`tr."transactionType"::text IN (${Prisma.join([...transactionType], ',')})`,
```

Comment:
`transactionType` might be a readonly array, and `Prisma.join` doesn't accept readonly arrays, so I need to convert it into a mutable array by the spread syntax `...`.